Detectors/CPV/simulation/src/RawWriter.cxx

```diff
@@ -40,7 +42,11 @@ void RawWriter::init()
     mRawWriter->registerLink(link.feeId, link.cruId, link.linkId, link.endPointId, rawFileName.data());
   }
 
+  // Lm-L0 delay
+  mLM_L0_delay = o2::ctp::TriggerOffsetsParam::Instance().LM_L0;
```

Comment:
I don't see where you fetch the TriggerOffsetsParam from the CCDB, seem you are always using default in digi2raw while for decoding you use it from CCDB?

Reply:
Yes, it is correct. If you suggest how to fetch it using BasicCCDBManager then I'll insert it.

Reply:
Exactly like you do for `CPV/Calib/Gains`, using the timestamp from the hbfutils. Simply, for the ConfigurableParams you don't need to assign the result of `ccdbMgr.get<o2::ctp::TriggerOffsetsParam>("CTP/Config/TriggerOffsets")`

Reply:
thanks, got it!